cyhy/mailer/CyhyMessage.py

```diff
@@ -61,6 +61,8 @@ class CyhyMessage(ReportMessage):
 
 <p>The Cyber Hygiene scan results are attached for your review. Same password as before. (If this is your first report and you have yet to receive a password, please let us know.)</p>
 
+<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paper clip icon to the left of the attachment name.
+
```

Comment:
@jeffkause You also need to add this new verbiage (minus the HTML) to the `TextBody` section above:
https://github.com/cisagov/cyhy-mailer/blob/8a80b7066d3273188d01708c49721c9e6db4bbb5/cyhy/mailer/CyhyMessage.py#L32-L35

Reply:
I have completed the additional wording and completed a successful pytest.

Reply:
Don't forget to push up your changes!

Reply:
Pushed. TY

Reply:
Looks like the `TextBody` verbiage was included in https://github.com/cisagov/cyhy-mailer/pull/111/commits/c7fc84d1c68bfde772d4eae4ea2f8818a54ad460.

@jeffkause In the future, please:
* Mention the commit URL (https://github.com/cisagov/cyhy-mailer/pull/111/commits/c7fc84d1c68bfde772d4eae4ea2f8818a54ad460) when you respond to the conversation; this makes it easier to see where you addressed the concerns raised in the conversation
* Replace the default GitHub commit message ("Update CyhyMessage.py") with a more useful commit message (e.g. "Add text-only content matching HTML email content").